### PR TITLE
Add description check

### PR DESCRIPTION
**Pull request by skoshchi**

### CODE REVIEW

File: src/main/java/io/xstefank/wildlfy/bot/config/Format.java [hunk 1]
```diff
@@ -7,4 +7,7 @@
 //    @ConfigProperty(name = "title-check")
     @JsonProperty("title-check")
     public RegexDefinition titleCheck;
+
+    @JsonProperty("description")
```
Comment (xstefank) [inline src/main/java/io/xstefank/wildlfy/bot/config/Format.java:11]: unnecessary if the name is the same as the name of the field

File: src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package io.xstefank.wildlfy.bot.format.checks;
+
+import io.xstefank.wildlfy.bot.config.RegexDefinition;
+import org.kohsuke.github.GHPullRequest;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DescriptionCheck implements Check{
```
Comment (xstefank) [inline src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java:9]: we need checkstyle I see :)

File: src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    public DescriptionCheck(RegexDefinition description) {
+        if (description == null || description.pattern == null) {
+            throw new IllegalArgumentException("Input argument cannot be null");
+        }
+        pattern = description.pattern;
+        message = (description.message != null) ? description.message : DEFAULT_MESSAGE;
+
+        System.out.println(pattern);
+        System.out.println(message);
```
Comment (xstefank) [inline src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java:23]: remove souts

Comment (xstefank) [reply src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java:23]: either use log or not print

**Review by xstefank — CHANGES_REQUESTED** [review]
Good work, a few small issues only

File: src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java (hunk 2 above)
Comment (skoshchi) [reply src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java:9]: Can u help to install checkstyle on my idea tomorrow?

File: src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java (hunk 3 above)
Comment (skoshchi) [reply src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java:23]: I deleted it. In **TitleCheck** there is the same thing

File: src/main/java/io/xstefank/wildlfy/bot/config/Format.java [hunk 4]
```diff
@@ -7,4 +7,7 @@
 //    @ConfigProperty(name = "title-check")
     @JsonProperty("title-check")
     public RegexDefinition titleCheck;
+
+    @JsonProperty()
```
Comment (xstefank) [inline src/main/java/io/xstefank/wildlfy/bot/config/Format.java:11]: no annotation at all is needed

**Review by xstefank — CHANGES_REQUESTED** [review]

**Review by xstefank — APPROVED** [review]

**Pull request merged**